md_file_standards/q07_symbolic-input/q07_symbolic-input2-Trig.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Symbolic Input 2 - Trig
+topic: example
+author: Michael Kudla
+source: original
+template_version: 0.1
+outcomes:
+tags:
+- quiz
+- homework
+assets:
+server: |
+  import prairielearn as pl
+  from sympy import *
+
+  #Note. One may be tempted to "import sympy" instead of "from sympy import *". This will work with simple multiplication/division, addition, and subtraction however this makes the sin and cos require sympy.sin and sympy.cos, which is messy. 
+  #Of course, if you are using "from numpy import *" as well, this will may cause errors, so use caution.
```

Comment:
This is exactly why we should *not* do `from sympy import *`. 

From [PEP8](https://www.python.org/dev/peps/pep-0008/) guide:

> Wildcard imports (from <module> import *) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools. There is one defensible use case for a wildcard import, which is to republish an internal interface as part of a public API (for example, overwriting a pure Python implementation of an interface with the definitions from an optional accelerator module and exactly which definitions will be overwritten isn't known in advance).

If you want, we can do `import sympy as sp` - that's only three extra letters (similar to `import numpy as np`)

Reply:
Yup, agree.